### DISCUSSION TITLE
Hypocrisy from laravel team in regards to lumen and laravel?

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#50547] [General]
Hi,
After watching this https://www.youtube.com/watch?v=lQSEBvxuXiU&ab_channel=LaravelNews from here https://laravel-news.com/laravel-11 and the release of laravel 11 corroborated with 
![image](https://github.com/laravel/framework/assets/82870454/7904dc78-1483-42a8-8874-aa41ad3e6922)
 I am wondering why (if lumen is to be deprecated) laravel 11 treats "api" routes as not used by the majority.... mind-blowing....
Is laravel becoming a framework oriented towards monoliths?

**Comment by martinbean** [comment]
@marius-mcp I don’t really understand where you think the “hypocrisy” is?

As the note says, there’s been a number of performance improvements to the PHP language itself that the performance benefit Lumen used to give is now almost non-existent versus a regular Laravel app.

As for API routes, no one is stopping you from using them in Laravel 11 applications. You’re free to define them: https://laravel.com/docs/11.x/routing#api-routes

**Reply by ghost** [reply]
@martinbean I agree with what you said and I know all that.

The feeling I'm getting is that when in need of an api framework you would choose lumen...
But wait...
The team behind it recommends laravel...
But wait again...
Laravel seems to not be focused on apis...

You get my point?

PS thanks for the correction.
